crud/common/utils.lua

```diff
@@ -1177,6 +1194,61 @@ function utils.storage_info_on_storage()
     return {status = "running"}
 end
 
+--- Initializes a storage function by its name.
+--
+--  It adds the function into the global scope by its name and required
+--  rights to a vshard storage user.
+--
+--  @function init_storage_call
+--
+--  @param string name a name of the function.
+--  @param function func the function.
+--
+--  @return nil
+function utils.init_storage_call(name, func)
+    dev_checks('string', 'function')
+
+    rawset(_G['_crud'], name, func)
+    name = utils.get_storage_call(name)
+
+    if type(box.cfg) ~= 'table' or box.cfg.read_only then
+        -- If the storage is not configured yet then it could be
+        -- a unit-test or something else.
```

Comment:
To be honest, I think running `init` on a non-configured instance should throw an error since we require instance to be configured. `ro` skip should be fine.